README.rst

```diff
@@ -104,6 +104,18 @@ Additionally, to report any concerns or
 
 |
 
+----------------------
+Benchmarking
+----------------------
+
+The performance of QAT can be measured using :code:`poetry run pytest --benchmark-only`, which runs our pre-defined benchmarks.
+To compare to main, checkout the main branch and run :code:`poetry run pytest --benchmark-only --benchmark-save="benchmark"`. 
```

Comment:
To avoid confusion it would be good to use the _standard_ variable notation here: `--benchmark-save="<benchmark-name."` unless it it explicitly the string `benchmark` that needs to be provided.

Also would be good to have a link to the docs for further reading.